Automattic/wp-calypso
# FSE: Update Homepage Posts block branding

#### Changes proposed in this Pull Request

Update the block name as per paYJgx-li-p2. To make the connection between visual side and code more clear, also update all related directory names.

#### Testing instructions

* Build FSE plugin based, based on its readme instructions
* Try this plugin on your WP installation
* In the UI, the block should be visible as "Blog Posts"
* When viewing the post source, it should show up as `a8c/blog-posts`
* Functionality should be untouched and block is expected to work both in editor and on the frontend
